ruby/range-cover-vs-include.md

```diff
@@ -0,0 +1,18 @@
+# Range cover? vs include? method
+
+The `cover?` method yields better performance than `include?`,
+however there are subtle differences to when one should be used over other.
```

Comment:
Running a simple benchmark with [readygo] and including the output could be interesting.

[readygo]: https://github.com/garybernhardt/readygo